paper/paper.md

```diff
@@ -1,158 +1,74 @@
+biohackathon_url:   "[https://biohackathon-europe.org/](https://www.swat4ls.org/workshops/leiden2024/hackathon/)"
+biohackathon_location: "Leiden, The Netherlands, 2024"
+group: Group 7
 # URL to project git repo --- should contain the actual paper.md:
-git_url: https://github.com/biohackrxiv/publication-template
+git_url: https://github.com/biohackathon-swat4hcls/swat4hcls-2024--publishing-fair-datasets-from-bioimaging-repositories.git
 # This is the short authors description that is used at the
 # bottom of the generated paper (typically the first two authors):
-authors_short: First Author \emph{et al.}
+authors_short: S. Dvoretskii, P. Schader, M. Nolden, J. Moore
 ---
 
 
-# Introduction
+# **Introduction**
 
-As part of the BioHackathon Europe 2023, we here report...
+During the SWAT4HCLS 2024 hackathon, we formed a group of participants from various research initiatives and consortia in Bioimaging and Medical Imaging, like the German Biobanking e.V, the Helmholtz Metadata Collaboration (HMC) Hub Health, the Division of Medical Image Computing at the German Cancer Research Center (DKFZ) and the EU Horizon project CCE\_DART (Data Rich Clinical Trials).
 
+With 2024 being the 10-year anniversary of FAIR data, our discussion quickly came to revolve around the state of FAIR access to Bioimaging data, and ideas how to improve it. Within the research bioimaging community, the Open Microscopy Environment (OME) Consortium has been working to standardize access to large-scale imaging data. The OME consortium is a long-term effort which has produced rich software and standards (\>=2000). Imaging Data Resource (IDR) has appeared as an annotation rich imaging portal in 2016\. As of today, it provides access to 129 Studies with more than 400TB data in more that 14 million images, continuously growing. (\[@usesMethodIn:Williams2017\]). 
 
+The FAIRness of data in IDR, like most other research data repositories, could be further improved. There are other resources that we would like to connect to, including those outside of the bioimaging domain. RDF is the best mechanism for that connection (\[@citesAsPotentialSolution:moore\_2024\_10687659\]). But a catalog is needed to make the produced RDF triples discoverable.
 
-# Bioimaging dataset aggregation projects
+To facilitate multi-centric research on medical imaging data the Kaapana framework (\[@usesMethodIn:scherer2020joint\]) was created. It provides an open-source framework based on cloud native technologies enabling the large-scale analysis of medical imaging data as well as sharing analysis methods across installations. Primary data as well as analysis results can be stored in the platform and metadata for example from DICOM images is extracted to support cross-center definition and curation of imaging patient cohorts.  Multi center data discovery, like in the DART project of Cancer Core Europe (CCE) also requires approaches for managing and sharing catalogs of metadata. .
```

Comment:
At the beginning of this paragraph, it might make sense to describe the history of your group(s).

Reply:
Thanks! I will find the brief history of MIC and HMC Hub Health and insert them here.